### PR TITLE
ARTEMIS-1438 Upgrade hawtio to latest (1.5.4)

### PR DESCRIPTION
**Pull request by michaelandrepearce**
Set hawtio to 1.5.4
Exclude diagnostics tab to avoid Flight Recorder being exposed.
Update jolokia to latest to match hawtio
Update current documentation links to artemis site instead of github causing blank load if docs are not local built.

### CODE REVIEW

**Comment by michaelandrepearce** [comment]
@howardgao found a way to hide the tab via js for new plugin thats in hawtio 1.5.1 and above, to avoid issue of exposing it in the console, so we should be good now just bumping up versions going forwards.

Also i bumped hawtio to 1.5.4 latest instead of 1.5.3 which was in your PR.

**Comment by clebertsuconic** [comment]
What would be the problem on having flight record exposed? it would help me a lot when debugging / profiling?

**Comment by michaelandrepearce** [comment]
It was on the original mail thread on adding the console, to remove Dan raised it is priority commercial feature and shouldn’t be in Apache console

**Comment by michaelandrepearce** [comment]
@clebertsuconic see here the original http://activemq.2283324.n4.nabble.com/DISCUSS-Removing-the-Web-Console-tp4717136p4728220.html 

It is why to get console in, at the time I just downgraded to hawtio 1.5.0 as it didn’t have a diagnostic plugin. 

I’m personally not opposed to exposing flight control, but as it was something in the initial reviews to ensure was removed I’d rather not ruffle feathers quite so soon maybe in a few more releases ;)

File: pom.xml [hunk 1]
```diff
@@ -604,7 +604,7 @@
            <groupId>org.jolokia</groupId>
            <artifactId>jolokia-war</artifactId>
            <type>war</type>
-           <version>1.3.5</version>
+           <version>1.3.7</version>
```
Comment (jbertram) [inline pom.xml]: Seems like we should move the &lt;jolokia-version> property from artemis-hawtio/pom.xml into here and then use that in both places so it's easier to keep in sync.

Comment (clebertsuconic) [reply pom.xml]: +1

Comment (clebertsuconic) [reply pom.xml]: that's an easy change

Comment (michaelandrepearce) [reply pom.xml]: I see this got merged in overnight (uk) , I’ll raise a new pr to address this, agreed best to have one place for versions

**Pull request closed without merging**